src/sisl/_core/geometry.py

```diff
@@ -3733,6 +3737,11 @@ def _orbital_values(self, grid_shape: tuple[int, int, int]):
         # supercell indices. Here we revert that
         ISC -= np.floor(self.fxyz[IA]).astype(int32)
 
+        # Don't consider atoms that are outside of the geometry's auxiliary cell.
+        if truncate_with_nsc:
+            mask = ~(abs(ISC) > self.nsc // 2).any(axis=1)
```

Comment:
why not `mask = (abs(ISC) <= self.nsc // 2).any(axis=1)`?

Reply:
Because that would mean "there is an axis that is within the constrains of `nsc`", and what I want is "there is no axis that is outside of the constraints of `nsc`".

Maybe I should use parenthesis to clarify it:

```python
~((abs(ISC) > self.nsc // 2).any(axis=1))
```

Reply:
So actually I could do `(abs(ISC) <= self.nsc // 2).all(axis=1)`

Reply:
sorry, `(abs(ISC) <= self.nsc // 2).all(axis=1)`?

Reply:
changed

Reply:
same code, same time, nice ;)